test/math_test.py

```diff
@@ -1377,15 +1132,15 @@ def testConstructionKeywords(self):
         self.assertEqual(v.z, 3.0)
 
     def testConstructionMissing(self):
```

Comment:
Is there a reason this test doesn't just call those Vector3 creation expressions inline with the asserts?

Reply:
Well... we can use a lambda to make it inline, but either way we need to pass a function-like object to `assertRaises`

Reply:
A function like object like the Vector3 constructor? :eyes:

Reply:
Ah... `assertRaises` would forward args and kwargs to the constructor